.github/workflows/deploy.yaml

```diff
@@ -20,7 +20,7 @@ permissions:
 # in-progress and latest queued. However, do NOT cancel in-progress runs as we
 # want to allow these production deployments to complete.
 concurrency:
-  group: "pages"
+  group: ${{ github.ref == 'refs/heads/main' && 'pages-main' || github.run_id }}
```

Comment:
Looks like the comment associated with this needs to be updated as it's now incorrect.